decksite/templates/entry.mustache

```diff
@@ -1 +1 @@
-<li><a href="{{card.url}}" class="{{^card.pd_legal}}illegal{{/card.pd_legal}} {{#card.bug_desc}}bugged{{/card.bug_desc}} card">{{n}} {{name}}</a></li>
+<li><a href="{{card.url}}" class="{{^card.pd_legal}}illegal{{/card.pd_legal}} {{#card.bug_desc}}bugged{{/card.bug_desc}} card" {{#card.bug_desc}}data-bug-desc="{{card.bug_class}}: {{card.bug_desc}}"{{/card.bug_desc}}>{{n}} {{name}}</a></li>
```

Comment:
We could write out the bug element with a title attribute here and the existing tooltip-adding code would add the tooltip and get rid of the need for initBugIcons?

Reply:
I think you are right. I was obssessed with creating everything in js, I'll try to do as you say here.